tee-worker/ts-tests/integration-tests/identity-direct-invocation.test.ts

```diff
@@ -120,6 +121,15 @@ describe('Test Identity (direct invocation)', function () {
         */
         assert.isTrue(res.do_watch.isFalse);
         assert.isTrue(res.status.asTrustedOperationStatus[0].isInvalid);
+        assertWorkerError(
+            context,
+            requestIdentifier,
+            (v) => {
+                assert.isTrue(v.isLinkIdentityFailed);
```

Comment:
:nail_care: for easier debugging, either go with

```
assert.equal(v.type, 'LinkIdentityFailed')
```

or print out the actual type in the message:

```
assert.isTrue(v.isLinkIdentityFailed, `expected LinkIdentityFailed, received ${v.type} instead`)
```